### ISSUE TITLE
Nextdoor for school districts of Houston (HISD, FBISD, etc.)

### ISSUE THREAD

**Issue by xpturk** [sketch-city/project-ideas#120]
Recently, I attended to a GT program meeting in FBISD and I've listened several real-world experiences and great ideas from the parents. That triggered this idea: 

_Parents need to exchange experiences and ideas, socialize with other parents, discuss common issues of their children and make suggestions to their districts. For that reason, we should give them an online platform which can be named as **"Nextdoor for Parents"** or **"Nextdoor for Schools/Districts"**_

The app lists districts of Greater Houston, the schools of those districts and people can sign up to those lists using their latest utility bills or driver licenses just as in Airbnb. If they live in the district or they are teachers in that district they can subscribe to district channel. 

Parents of a school or a class can use this app to socialize more and discuss their children's common issues. They can organize parent coffee nights, "dad's club"s in the school, principals can use this platform to request an urgent help from volunteers for rainy days etc. It can also include polls as well. 

What do you think?

**Comment by ChristianArredondo** [comment]
Like this? https://www.edmodo.com/

But less focused on schoolwork?

**Comment by xpturk** [comment]
@ChristianArredondo Thanks for the feedback. I checked the website and the app. If I'm not wrong edmodo is a kind of parent-teacher-student collaboration program and it has so many features but doesn't have my core functionality. 

What I suggested is a "very simple" socialization app that is just for parents, teachers and districts to exchange ideas, collect suggestions and comments, polls etc. 

It will have only one purpose:  To increase communication between parents, teachers and their districts, and no more.
